powerstore/resource_storagecontainer.go

```diff
@@ -154,7 +163,73 @@ func (r resourceStorageContainer) Read(ctx context.Context, req tfsdk.ReadResour
 
 // Update resource
 func (r resourceStorageContainer) Update(ctx context.Context, req tfsdk.UpdateResourceRequest, resp *tfsdk.UpdateResourceResponse) {
+	log.Printf("Started Update")
+
+	// Get plan values
+	var plan models.StorageContainer
+	diags := req.Plan.Get(ctx, &plan)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	// Get current state
+	var state models.StorageContainer
+	diags = req.State.Get(ctx, &state)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	// a worksround
+	// currently a bug on powerstore, not accepting PATCH call for same values
```

Comment:
Not a good comment.
Also are we sure it is a bug and something not done deliberately by Powerstore team.
I think you should seek the reason for the same before mentioning it a bug.

Reply:
removing the term "bug"